.github/workflows/ray.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Ray
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - "lineapy/plugins/jinja_templates/*"
+      - "lineapy/plugins/ray_pipeline_writer.py"
+      - "lineapy/plugins/base_pipeline_writer.py"
+      - "tests/test_pipeline_run_ray.py"
+  pull_request:
+    paths:
+      - "lineapy/plugins/jinja_templates/*"
+      - "lineapy/plugins/ray_pipeline_writer.py"
+      - "lineapy/plugins/base_pipeline_writer.py"
+      - "tests/test_pipeline_run_ray.py"
+      - "ray-requirements.txt"
+jobs:
+  ray:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+          submodules: "recursive"
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
```

Comment:
only version 3.9 ok?

Reply:
Its the same for airflow, I suppose it's ok for this suite of tests since its pretty slow as well.